bot/exts/valentines/valentine_zodiac.py

```diff
@@ -29,13 +37,127 @@ def load_json() -> dict:
             zodiacs = load(json_data)
             return zodiacs
 
-    @commands.command(name="partnerzodiac")
-    async def counter_zodiac(self, ctx: commands.Context, zodiac_sign: str) -> None:
+    def zodiac_sign_verify(self, zodiac: str) -> discord.Embed:
```

Comment:
this method can be a staticmethod

Reply:
i need self in this, so i cant change this to static method